src/main.cpp

```diff
@@ -656,6 +672,26 @@ int main(int argc, char *argv[])
                 params.codec = optarg;
                 break;
 
+            case 'r':
+                params.framerate = atoi(optarg);
+                break;
+
+            case 'x':
+                params.pix_fmt = optarg;
+                break;
+
+	    case 'C':
+	        params.acodec = optarg;
+	        break;
+
+	    case 'R':
+	        params.sample_rate = atoi(optarg);
```

Comment:
We should at least print what values we have parsed for sample_rates, because otherwise if the user makes a typo they will have a hard time spotting it.